### ISSUE TITLE
Request organizations

### ISSUE THREAD

**Issue by CashDesk** [osTicket/osTicket#5356]
Hi, I have the following request for organizations (users, organizations).
Can the option be added to create a filter with open / closed tickets in this screen?
Can the option be added by clicking on the column name to do a sort?

**Comment by JediKev** [comment]
@CashDesk 

Yes, we have this on our Feature Request list already; revamping the User Directory Ticket queue, Organization Ticket Queue, etc. to be _at least_ sortable. We also hope to make them customizable to match the main ticket queues so agents can configure how they want to view User/Organization tickets.

Cheers.

**Comment by CashDesk** [comment]
@JediKev Great to hear that. Thanks!

**Comment by mlipok** [comment]
@CashDesk could you look here:
https://github.com/osTicket/osTicket/issues/5263
and chceck if this is a handy feature for you ?
and also could could you check if you have the same issue ?

**Comment by CashDesk** [comment]
@mlipok , As far as I can see my request was another one. I like to manage all tickets by the organization. What I understand from your request is that you like to manage all tickets by a user.

**Comment by Hardjanto12** [comment]
can you add option to add column on user directory please
